### PR TITLE
rabbitmq.com

### LINKED ISSUE #686: +rabbitmq (441/548)
Formulae: https://formulae.brew.sh/formula/rabbitmq

### PR DESCRIPTION
**Pull request by and-ri**
closes #686

### CODE REVIEW

File: projects/rabbitmq.com/package.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  strip-components: 1
+versions:
+  github: rabbitmq/rabbitmq-server
+warnings:
+  - vendored
+dependencies:
+  erlang.org: '*'
+build:
+  dependencies:
+    python.org: ~3.11
```
Comment (jhheider) [inline projects/rabbitmq.com/package.yml]: What is using python?

File: projects/rabbitmq.com/package.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - bin/rabbitmqctl
+  - bin/rabbitmq-defaults
+  - bin/rabbitmq-diagnostics
+  - bin/rabbitmq-env
+  - bin/rabbitmq-plugins
+  - bin/rabbitmq-queues
+  - bin/rabbitmq-server
+  - bin/rabbitmq-streams
+  - bin/rabbitmq-upgrade
+  - bin/vmware-rabbitmq
```
Comment (jhheider) [inline projects/rabbitmq.com/package.yml]: Should be fine to leave everything alone in sbin

File: projects/rabbitmq.com/package.yml (hunk 1 above)
Comment (and-ri) [reply projects/rabbitmq.com/package.yml]: I looked up this dependency here: https://github.com/Homebrew/homebrew-core/blob/e0e87ed974ed800f73dfe7ebcc49af030b088a16/Formula/r/rabbitmq.rb 😇

File: projects/rabbitmq.com/package.yml (hunk 2 above)
Comment (and-ri) [reply projects/rabbitmq.com/package.yml]: Oh, I thought that `provides` must be in `bin/` dir

Comment (jhheider) [reply projects/rabbitmq.com/package.yml]: We have others with sbins. Unbound comes to mind.

**Comment by and-ri** [comment]
hmm, why was everything ok in https://github.com/pkgxdev/pantry/pull/3575/commits/7db6780aeb049d1f353c62dc0c0d2506d888e80b (https://github.com/pkgxdev/pantry/actions/runs/6449619143)
strange 🤔

**Comment by jhheider** [comment]
> hmm, why was everything ok in [7db6780](https://github.com/pkgxdev/pantry/commit/7db6780aeb049d1f353c62dc0c0d2506d888e80b) (https://github.com/pkgxdev/pantry/actions/runs/6449619143) strange 🤔

is it possible you were making dangling symlinks? `ln` will create a symlink to something that doesn't exist.

File: projects/rabbitmq.com/package.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+test:
+  env:
+    RABBITMQ_MNESIA_BASE: $PWD/var/lib/rabbitmq/mnesia
+  script:
+    - rabbitmqctl --version | grep {{version}}
+    - rabbitmq-server -detached && sleep 15
```
Comment (jhheider) [inline projects/rabbitmq.com/package.yml:36]: you probably need a `killall rabbitmq-server` before this, in case a failed run has left the server running.

**Pull request merged**